### PR TITLE
Add list of interesting accounts in local dev login page

### LINKED ISSUE #16084: Add list of interesting accounts in local dev login page
For people like me who have a bunch of different accounts locally with different sets of permissions, it's always a pain to remember which account has what and what email address I used.

If `settings.DEBUG` and `settings.USE_FAKE_FXA_AUTH` are `True` we have a completely fake login page anyway, so we should suggest interesting accounts there, for instance listing those that have any special permissions, making it easier to juggle through different accounts locally.

### PR DESCRIPTION
**Pull request by diox**
Fixes #16084

### CODE REVIEW

File: static/js/zamboni/users.js [hunk 1]
```diff
@@ -3,6 +3,12 @@ $('#admin-login').click(function () {
   window.location = $(this).attr('data-url');
 });
 
+$('#fake_fxa_authorization .accounts a').click(function (e) {
+  $('#id_email').prop('value', $(this).prop('href').replace('mailto:', ''));
```
Comment (willdurand) [inline static/js/zamboni/users.js:7]: I was wondering how a `mailto` would submit a form... Maybe it should not be `mailto` but a data attribute?

**Review by willdurand — APPROVED** [review]
r+wc

File: static/js/zamboni/users.js (hunk 1 above)
Comment (diox) [reply static/js/zamboni/users.js:7]: Maybe... it was already in the text and I prefer having actual links but maybe I don't need to...

Comment (willdurand) [reply static/js/zamboni/users.js:7]: (it doesn't matter much and I dont have a better idea right now fwiw)

**Pull request merged**